src/index.html

```diff
@@ -11,8 +11,50 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section>
+      <div class="comp">
+        <div class="comp__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="comp__product-code">
+          Product code: 195434
+        </div>
+        <div class="comp__rate">
+          <div class="comp__rate--stars comp__rate--stars--4">
+            <div class="comp__rate--stars__star"></div>
+            <div class="comp__rate--stars__star"></div>
+            <div class="comp__rate--stars__star"></div>
+            <div class="comp__rate--stars__star"></div>
+            <div class="comp__rate--stars__star"></div>
```

Comment:
The class `comp__rate--stars__star` is not correctly following BEM naming conventions. The double underscore is used for elements, and it should be `comp__rate--star` if it's a modifier, or `comp__rate__star` if it's an element of `comp__rate`.